evidence/pool.go

```diff
@@ -49,10 +49,9 @@ type Pool struct {
 // NewPool creates an evidence pool. If using an existing evidence store,
```

Comment:
This file diff contains linting updates only.

Reply:
Thanks for the heads-up! 😅